apps/studio/app/api/ai/sql/suggest/route.ts

```diff
@@ -0,0 +1,132 @@
+import { OpenAIStream, StreamingTextResponse } from 'ai'
+import { codeBlock, oneLine, stripIndent } from 'common-tags'
+import { NextRequest } from 'next/server'
+import OpenAI from 'openai'
+
+export const runtime = 'edge'
+
+const openAiKey = process.env.OPENAI_KEY
+
+/**
+ * This route has to be with an edge runtime to support streaming. This route is not using the
+ * apiWrapper because it's not working with edge API routes (different params) but the route doesn't
+ * require authentication.
+ */
+export async function POST(request: NextRequest) {
```

Comment:
just for consistency

```suggestion
export async function handler(request: NextRequest) {
```

Reply:
It has to be POST because this is a `/app` route handler.